criu/mount.c

```diff
@@ -2014,7 +2017,10 @@ static int fetch_rt_stat(struct mount_info *m, const char *where)
 static int do_simple_mount(struct mount_info *mi, const char *src, const
 			   char *fstype, unsigned long mountflags)
 {
-	return mount(src, mi->mountpoint, fstype, mountflags, mi->options);
+	int ret = mount(src, mi->mountpoint, fstype, mountflags, mi->options);
+	if (ret)
+		pr_perror("mount(%s, %s)", src, mi->mountpoint);
```

Comment:
Can you please add mi->mnt_id as mountpoints can be the same for different mounts.

Reply:
Done